templates/script/location-trips.html

```diff
@@ -1,5 +1,4 @@
 <script>
-	var current_location = {{ user.current_location or [] }}
 	var locations = [
```

Comment:
Not sure why we have 4 spaces in this file while it's 2 spaces all around.
I not this is out of this PR, just noticing.